### ISSUE TITLE
Outlook Calendar Entries wont delete from Google Calendar

### ISSUE THREAD

**Issue by freewuns** [phw198/OutlookGoogleCalendarSync#1788]
Version 2.10.1.0

I can create calendar entries in Outlook and they show in my Google calendar on Android.
I can create calendar entries in Google on Android and they show in my Outlook calendar.
When I delete calendar entries in Outlook the entries are deleted in Google calendar.
When I delete calendar entries in Google calendar they are NOT deleted in Outlook calendar.

Clearly syncing is working as entries are shared. It also doesnt matter where the calendar entry originates, the deleted entry in Google wont delete in Outlook.

Using Win 11 and Outlook 2019.

Any advice appreciated.

**Comment by phw198** [comment]
You don't have "Disable deletions" checked perchance, do you?

**Comment by freewuns** [comment]
![pic1](https://github.com/phw198/OutlookGoogleCalendarSync/assets/158232466/baf87504-07bd-4b2b-a3d0-445db0309094)
![pic2](https://github.com/phw198/OutlookGoogleCalendarSync/assets/158232466/6bff8441-e1bc-4a5e-8a2d-d0837cdc72a3)
![pic3](https://github.com/phw198/OutlookGoogleCalendarSync/assets/158232466/5954b08f-2bdd-4169-8ca6-fb67123381f7)
> You don't have "Disable deletions" checked perchance, do you?

No, havent selected.

**Comment by phw198** [comment]
You have "for items created by sync in Google....set privacy to Private", and then are excluding Private items from sync.

**Comment by freewuns** [comment]
> You have "for items created by sync in Google....set privacy to Private", and then are excluding Private items from sync.

Thank you. That seems to have worked :)

When I went to Win11 from 7 I lost my settings for OGCS but they are saved now!

Much appreciate the help!

**Comment by freewuns** [comment]
Hi Paul,

Still having issues deleting in Android and it reflecting in the Outlook Calendar. This may help....

If I create a new calendar event either single or a recurring, Outlook updates fine. I can even delete the single event or recurring in Android and Outlook will do the same.

The problem happens when any event on the calendar reaches it's time for the alert. I either snooze, let it expire or just close it in Android.

When I go to delete it after the event has started, thats when it wont delete in Outlook. But I can still delete the series if its recurring event.

So in Summary, the deletions will sync if I delete the event before it occurs but they wont delete in Outlook if I delete them in Android after the event.

Any ideas? I have tried multiple settings changes without luck.

Cheers,

Karl

**Comment by phw198** [comment]
You'd need to post your log file and specify which item you have deleted in Google that won't delete in Outlook.

Also, please confirm the item is deleted in https://calendar.google.com first too and the issue isn't with your phone app.

**Comment by freewuns** [comment]
Hi Paul,
I deleted some of my recurring calendar events on both Google and Outlook and input them again. Seems to have done the trick so far.

One of them must have been causing an issue. I tried to understand the log file without much luck.

Thank you :-)
